app/workers/deliver_email_worker.rb

```diff
@@ -14,8 +14,19 @@ def self.queue_for_priority(priority)
   sidekiq_options retry: 3, queue: queue_for_priority(:low)
 
   def perform(email_id)
+    start = Time.now.utc
+
     email = Email.find(email_id)
     DeliverEmail.call(email: email)
+
+    duration = (start - Time.now.utc) * 1000.0
```

Comment:
is this going to return a negative number?
in which case, will https://github.com/alphagov/email-alert-api/pull/233/files#diff-69752d22c8871b0f2ec09d2de4ef14aeR6 ever be true for a positive `minimum_duration_in_milliseconds`?

Reply:
Back to front... Time is not one of my strong points. 🤕 I've got rid of that stuff now.